### ISSUE TITLE
Ethos 1.6.0EU RAM overflow after opening several .bmp files 

### ISSUE THREAD

**Issue by philippe-warnotte** [FrSkyRC/ETHOS-Feedback-Community#4943]
Kavan V20 Drives: RADIO 8Go RAM 8Mo.
By successively opening several BMP files (+/- 650KB) using "System/file manager", 
the available RAM decreases! After 4 files BMP readings, only 1Mo RAM remains available.
Afterwards that, previewing and opening  BMP is no longer possible.(image or screenshot).
RAM memory does not empty when closing the file.
The only way to recover RAM is to power off/on the transmitter.
The model used does not matter. Only 1 script is used: stabilizer config.
When the radio is turned ON, 4.9Mo are available.
Perhaps you should use a "Swapfile" like Microsoft Windows...

**Comment by WhiskeyGolf** [comment]
I have the same Problem on opening Screenshots.

**Comment by mgsanna** [comment]
The same behavior on X20PRO AW, the system ram is not emptied on exiting and not recovered... maybe it could be a good idea to purge ram after closing items like bitmaps or Screenshots.
I think that scripts are not involved in the process

**Comment by WhiskeyGolf** [comment]
Yes, the risk of running into emergency mode would then also be lower.

**Comment by philippe-warnotte** [comment]
This is a BUG similar to #4870

**Comment by bsongis-frsky** [comment]
I am on X20-PRO and I have ~30 screenshots. I can look at all of them without a problem. The RAM available stays at 2.8MB

**Comment by bsongis-frsky** [comment]
Ah. Just after writing the comment above I succeeded to reproduce the problem :)

**Comment by bsongis-frsky** [comment]
Thanks for this report. I have understood the issue wrongly, I thought it was the bitmap preview which leaked memory. But it is the Bitmap Viewer. Huge memory leak indeed!

**Comment by philippe-warnotte** [comment]
If I may: I think it's the same BUG as #4870. 
You had found a solution to no longer go to E.M by keeping 1 Mo RAM available. 
But the memory leak is still there as you see.

**Comment by bsongis-frsky** [comment]
Indeed this one is perhaps the preliminary part for reaching #4870
